open-vm-tools-centos/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos 
+
+
+
+LABEL summary="The open-vm-tools guest agent" \
+      io.k8s.description="The open-vm-tools agent is providing information about the virtual machine and allows to restart / shutdown the machine via VMware products. This image is intended to be used with virtual machines running Centos Atomic Host." \
+      io.k8s.display-name="open-vm-tools guest agent" \
+      architecture="x86_64" \
+      BZComponent="open-vm-tools" \
+      maintainer="davis phillips <dphillip@redhat.com>"
+
+ENV SYSTEMD_IGNORE_CHROOT=1
+
+RUN yum -y --disablerepo=\* --enablerepo=rhel-7-server-rpms install yum-utils && \
+  yum-config-manager --disable \* && \
+  yum-config-manager --enable rhel-7-server-rpms && \
+  yum clean all
```

Comment:
I wouldn't think it would be needed. ```open-vm-tools``` is available in [CentOS 7](http://mirror.centos.org/centos/7/os/x86_64/Packages/). However I wasn't able to find ```open-vm-tools-deploypkg ```.